packages/asset-swapper/src/contracts.ts

```diff
@@ -0,0 +1 @@
+export { ERC20BridgeSamplerContract } from '../generated-wrappers/erc20_bridge_sampler';
```

Comment:
this gets around an `Didn't find an exportPath for` error

Reply:
This is so weird. Where do you see this error?